pootle/apps/import_export/utils.py

```diff
@@ -49,8 +50,15 @@ def import_file(file, user=None):
                                 "Project/Language? (%s)", (file.name, e)))
 
     try:
+        tp = store.translation_project
+        allow_add_obsolete = ((tp.project.code == 'terminology' or
```

Comment:
@Taras  one thing im not sure about re terminology projects is whether users should be allowed to obsolete or just add - we should check this with @dwaynebailey